data/blitz_berliner_weisse.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "Blitz Berliner Weisse",
+  "tagline": "Berliner Fruit Beer.",
+  "first_brewed": "07/2013",
+  "description": "Our sour recipe for all fruit Blitz beers uses a process called kettle souring. In this we steep a bag of malt in the wort to allow the bacteria to grow in it.",
+  "abv": 3,
+  "ibu": 8,
+  "target_fg": 1040,
+  "target_og": 1007,
+  "ebc": 9,
+  "srm": 4.5,
+  "ph": 3.2,
+  "attenuation_level": 82.5,
+  "volume": {
+    "value": 20,
+    "unit": "liters"
+  },
+  "boil_volume": {
+    "value": 25,
+    "unit": "liters"
+  },
+  "method": {
+    "mash_temp": {
+      "temp": {
+        "value": 65,
+        "unit": "celsius"
+      },
+      "duration": 60
+    },
+    "fermentation": {
+      "temp": {
+        "value": 19,
+        "unit": "celsius"
+      }
+    },
+    "twist": {
+      "technique": "Before main fermentation steep a bag of malt in the wort for 48 hours. Re boil the wort before pitching in the yeast for the main fermentation"
```

Comment:
I have now removed tehcnique and instead flattened it to just a string value for `twist`, would you be cool to do the same